src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -541,12 +541,10 @@ public void validationErrorWhenRenewalPeriodForHoldsSpecifiedForFixedPolicy() {
       .withPickupServicePointId(servicePointsFixture.cd1().getId())
       .by(usersFixture.charlotte()));
 
-    Response response = loansFixture.attemptRenewal(500, smallAngryPlanet, rebecca);
+    Response response = loansFixture.attemptRenewal(422, smallAngryPlanet, rebecca);
 
-    assertThat(
-      response.getBody(),
-      is("Item's loan policy has fixed profile but alternative renewal period for holds is specified")
-    );
+    assertThat(response.getJson(), hasErrorWith(
+      hasMessage("Item's loan policy has fixed profile but alternative renewal period for holds is specified")));
```

Comment:
This change in status code suggests that the client should address this issue. Are you expecting the client to report / fix the issue with the policy?

Reply:
Hmm, good question. Actually the presence of the property should not prevent renewal, as far as I know. This is basically means that a fixed loan policy has properties from rolling policy.

Reply:
> Actually the presence of the property should not prevent renewal, as far as I know. This is basically means that a fixed loan policy has properties from rolling policy.

To me, the presence of that property suggests that the policy might be corrupt. I think that was the intention of this check, I tend to like to fail fast when I think something is misconfigured.

Reply:
Ok, so I leave it as it is.

Reply:
Ok